src/styles/landing.module.css

```diff
@@ -0,0 +1,153 @@
+* {
+  font-family: "Netflix Sans", "Helvetica Neue", Helvetica, Arial, sans-serif;
+}
+
+p {
+  color: rebeccapurple;
+  font-size: larger;
```

Comment:
you can create 1 folder called `styles` instead of styles you can have `variables` file and let's define all of your properties as a value such as Fonts, Colors, etc. and then inside of this file on the top you can import it and use it for example

`styles/variables.scss ->`
```
$red: #ff0000; -> ${valueName}: {value}
```

`landing.module.scss` ->
```
@import '${pathToVariavles}' -> ex '../../styles/variables'

// inside of ayn class
.exampleClass {
      color: $red;
}
```

**NOTE: All colors, fonts etc should be come from variables.scss**